tests/components/MCQ.test.ts

```diff
@@ -57,4 +57,21 @@ describe("MCQ.vue", () => {
       expect(renderedOption.text()).toBe(`${value.text}`);
     }
   });
+
+  test("Selecting an option", async () => {
+    const wrapper = mount(MCQ, {
+      props: {
+        title,
+        options,
+      },
+    });
```

Comment:
I'd be considering moving this into a function (`mountMCQ()`). It is starting to get repetative.